tests/ethereum/test_general.py

```diff
@@ -1076,7 +1075,7 @@ def test_event_forwarding_after_state_fork_during_message_call(self):
 
         contract Lib {
            function isSeven(uint a) public pure returns (bool) {
-               if (a == 7) {
+               if (a == 0x414243444546) {
```

Comment:
The function is still called `isSeven` but the value is now compared to 0x414243444546